### PR TITLE
Add models 

### PR DESCRIPTION
**Pull request by Alice932**

### CODE REVIEW

**Review by olehvavryniv — COMMENTED** [review]
Please remove `.DS_Store` files.

**Review by olehvavryniv — APPROVED** [review]

File: lesson5/db/schema.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# This file is auto-generated from the current state of the database. Instead
```
Comment (YuliiaManivska) [inline lesson5/db/schema.rb:1]: A lot of comments

**Review by YuliiaManivska — APPROVED** [review]

**Review by mariadenadel — APPROVED** [review]
Add .gitignore file for every repo

**Review by Nazarcheck — APPROVED** [review]
Well done)

**Pull request merged**